### DISCUSSION TITLE
doubt on installation

### DISCUSSION THREAD

**Discussion by thaynos** [microsoft/winget-cli#1017] [Q&A] [answered]
# Brief description of your issue
i installed app isntaller using the ms store link given in documentation but i dont see it in the search bar. i even tried the winget command in cmd but it doesnt work. in the windows app it shows i have installed it. i tried to install manually but i dont understand what to do with the admx and adml file. i dont have admin privileges and i am on a guest account. i also mentioned both cmd and search bar bug because i assume theyre corellated. im not al that experienced in coding or command line.
# Steps to reproduce

 given in desription

# Expected behavior

 i expected it to download power tools in cmd and show up in search bar. 

# Actual behavior

but it shows winget is not recognized as internal or external command and does not show up in the search bar. 

# Environment

```none
[winget --info]
Windows Package Manager version
Windows: Windows.Desktop version
Package: Microsoft.DesktopAppInstaller version

i did not understand what to do in this section

Any other software?
```

**Comment by denelon** [comment, marked as answer]
@thaynos in order to get the version of the App Installer with the Windows Package Manager from the Microsoft Store, you must be running on an Insider build of Windows, or have joined the Windows Package Manager Insider program. You can also [directly install](https://aka.ms/getwinget) the 1.0 release from GitHub.

You can check to see which version is installed via PowerShell with the following command:
`Get-AppxPackage microsoft.desktopappinstaller`

You should open PowerShell directly rather than executing it in Windows Terminal If you see the following error:
`Get-AppxPackage: The 'Get-AppxPackage' command was found in the module 'Appx', but the module could not be loaded. For more information, run 'Import-Module Appx'.`

You should open PowerShell directly rather than executing it in Command Prompt if you see the following error:
`The 'Get-AppxPackage' command was found in the module 'Appx', but the module could not be loaded. For more information, run 'Import-Module Appx'.
'Get-AppxPackage:' is not recognized as an internal or external command,
operable program or batch file.`

**Comment by denelon** [comment]
The output for `Get-AppxPackage microsoft.desktopappinstaller` contains the Version of the "DesktopAppInstaller". Any version greater than 1.11 contains the Windows Package Manager.

Example output:

```
PS C:\Users\denelon> Get-AppxPackage microsoft.desktopappinstaller


Name              : Microsoft.DesktopAppInstaller
Publisher         : CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US
Architecture      : X64
ResourceId        :
Version           : 1.11.11451.0
PackageFullName   : Microsoft.DesktopAppInstaller_1.11.11451.0_x64__8wekyb3d8bbwe
InstallLocation   : C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.11.11451.0_x64__8wekyb3d8bbwe
IsFramework       : False
PackageFamilyName : Microsoft.DesktopAppInstaller_8wekyb3d8bbwe
PublisherId       : 8wekyb3d8bbwe
IsResourcePackage : False
IsBundle          : False
IsDevelopmentMode : False
NonRemovable      : False
Dependencies      : {Microsoft.VCLibs.140.00.UWPDesktop_14.0.29231.0_x64__8wekyb3d8bbwe,
                    Microsoft.DesktopAppInstaller_1.11.11451.0_neutral_split.scale-100_8wekyb3d8bbwe}
IsPartiallyStaged : False
SignatureKind     : Store
Status            : Ok
```

**Reply by thaynos** [reply]
```

Name              : Microsoft.DesktopAppInstaller
Publisher         : CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US
Architecture      : X64
ResourceId        :
Version           : 1.4.3161.0
PackageFullName   : Microsoft.DesktopAppInstaller_1.4.3161.0_x64__8wekyb3d8bbwe
InstallLocation   : C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.4.3161.0_x64__8wekyb3d8bbwe
IsFramework       : False
PackageFamilyName : Microsoft.DesktopAppInstaller_8wekyb3d8bbwe
PublisherId       : 8wekyb3d8bbwe
IsResourcePackage : False
IsBundle          : False
IsDevelopmentMode : False
NonRemovable      : False
Dependencies      : {Microsoft.VCLibs.140.00.UWPDesktop_14.0.29231.0_x64__8wekyb3d8bbwe,
                    Microsoft.VCLibs.140.00_14.0.29231.0_x64__8wekyb3d8bbwe,
                    Microsoft.DesktopAppInstaller_1.4.3161.0_neutral_split.scale-100_8wekyb3d8bbwe,
                    Microsoft.DesktopAppInstaller_1.4.3161.0_neutral_split.language-fr_8wekyb3d8bbwe...}
IsPartiallyStaged : False
SignatureKind     : Store
Status            : Ok

```

i got this and i see its less than 1.11. isnt windows store supposed to download the latest version? i have downloaded it from windows store.
also i dont understand what to do with the appxbundle.

**Comment by thaynos** [comment]
also i think the reason why it never showed in search bar is because it never installed in the first place. but i always click on isntall and it shows gettign ready to install but then it just disappears.

**Reply by denelon** [reply]
@thaynos if you download the .appxbundle from the GitHub release, you "should" be able to double click and get it to install as long as you are on Windows 10 version 1809 or newer.
